src/Sources/MultiParticleSources.jl

```diff
@@ -0,0 +1,170 @@
+"""
+  Microphysical source functions involving interactions between particles
+  
+  Includes terms such as collisional coalescence
+"""
```

Comment:
This is nice how you break the integrals down to pieces. We should do the same in the other file too (but in a future PR).